GIT-VERSION-GEN

```diff
@@ -60,21 +74,21 @@ else
 	VN="$DEF_VER"
 fi
 
-dirty=$(sh -c 'git diff-index --name-only HEAD' 2>/dev/null) || dirty=
+dirty=$(sh -c "git -C '$SOURCE_DIR' diff-index --name-only HEAD" 2>/dev/null) || dirty=
```

Comment:
This change isn't kosher because single-quotes in `$SOURCE_DIR` aren't quoted. I don't see why we have to run`git` in `sh -c` in the first place. Just run it in `$()` like we do everywhere else, including the redirection.

Reply:
Good point, fixed.